sympy/stats/stochastic_process_types.py

```diff
@@ -759,3 +765,221 @@ def limiting_distribution(self):
         eqs.append(sum(wi) - 1)
         soln = list(linsolve(eqs, wi))[0]
         return ImmutableMatrix([[sol for sol in soln]])
+
+
+class BernoulliProcess(DiscreteTimeStochasticProcess):
+    """
+    The Bernoulli process consists of repeated
+    independent Bernoulli trials with the same parameter `p`.
+    It's assumed that the probability `p` applies to every
+    trial and that the outcomes of each trial
+    are independent of all the rest. Therefore Bernoulli Processs
+    is Discrete State and Discrete Time Stochastic Process.
+
+    Parameters
+    ==========
+
+    sym: Symbol/string_types
+    success: Event of success
+    failure: Event of failure
+    p: Proability of getting success
+
+    Examples
+    ========
+
+    >>> from sympy.stats import BernoulliProcess, P, E
+    >>> from sympy import Eq, Gt, Lt
+    >>> B = BernoulliProcess("B",p=0.7, success=1, failure=0)
+
+    >>> B.state_space
+    FiniteSet(0, 1)
+
```

Comment:
The blank spaces between doctests are not needed if you aren't adding any commentary.